paasta_tools/cli/schemas/kubernetes_schema.json

```diff
@@ -190,7 +190,124 @@
                     "type": "string"
                 },
                 "autoscaling": {
-                    "type": "object"
+                    "type": "object",
+                    "properties": {
```

Comment:
wow, we never had a schema for the autoscaling stuff?

Reply:
apparently not